events/scheme_events/stasis_trap_scheme/stasis_trap_outcome_events.txt

```diff
@@ -0,0 +1,400 @@
+﻿namespace = stasis_trap_outcome
```

Comment:
I suppose it's copypasted events so created scripted triggers, effects out of shared chunks of code.